pkg/coderefs/alias.go

```diff
@@ -1,15 +1,33 @@
 package coderefs
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/iancoleman/strcase"
 	"github.com/launchdarkly/ld-find-code-refs/internal/helpers"
 	"github.com/launchdarkly/ld-find-code-refs/internal/options"
+	"github.com/launchdarkly/ld-find-code-refs/internal/validation"
 )
 
-func generateAliases(flags []string, aliases []options.Alias) (map[string][]string, error) {
+func generateAliases(flags []string, aliases []options.Alias, dir string) (map[string][]string, error) {
```

Comment:
This function is a port of the `alias.Generate` function that used to be in the `options` package. The main difference is that the contents of files is no longer stored on the `Alias` itself.